src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeEventRepositoryTest.java

```diff
@@ -52,10 +53,10 @@ void should_save_and_read_events() {
         // then
         assertThat(eventsInDb)
             .hasSize(2)
-            .extracting(e -> tuple(e.id, e.envelopeId, e.type, e.errorCode, e.notes))
+            .usingElementComparatorIgnoringFields("createdAt")
             .containsExactlyInAnyOrder(
-                tuple(eventId1, envelopeId, event1.type, event1.errorCode, event1.notes),
```

Comment:
This probably shouldn't be any order anymore. We might also need to add a pause between event insertions.

Reply:
This is an existing test which is using `tuple`. I updated it to use `usingElementComparatorIgnoringFields`. I'll revert these changes if you want them in a separate PR